test/FormGroupSpec.js

```diff
@@ -46,6 +46,15 @@ describe('FormGroup', function() {
     assert.ok(ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'form-group-lg'));
   });
 
+  // This test case must come first, since the error only gets logged once.
```

Comment:
Per https://github.com/facebook/react/blob/a56d0d6dc58a973d103bfae7e7ef7c04226209c7/src/classic/element/ReactElementValidator.js#L273 each prop type warning only gets logged once. Therefore, if I put this test case after the next one, no warning would be thrown even if my code didn't work.